### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Add validation for inputs email and password
- When press button submit, the app navigate to Library

#### Galaxy S (or similar)
Screenshot_1622136678

#### Pixel 2/3 (or similar)
Screenshot_1622136878

#### Pixel 3XL (or similar)
Screenshot_1622136272

#### iPhone SE (or similar)
simulator_screenshot_E8748784-8758-4E9F-A663-4B35A8808273

#### iPhone 6/7/8 (or similar)
simulator_screenshot_6490E16A-C262-4202-B14C-9BA4E309C722

#### iPhone 11/12 Base/Pro/Pro Max (or similar)
simulator_screenshot_41CFE6A8-10B0-4D1A-B7D3-6CA90D5B1A9A

## Trello Card

https://trello.com/c/wNGpTlHK/30-n-102-validaci%C3%B3n-y-submit

### CODE REVIEW

File: wbooks/src/app/screens/Login/components/Form/index.tsx [hunk 1]
```diff
@@ -5,14 +5,23 @@ import styles from './styles';
 
 interface Props {
   title: string;
+  value: any;
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Login/components/Form/index.tsx]: replace any with appropriate type

File: wbooks/src/app/screens/Login/index.tsx [hunk 2]
```diff
@@ -1,16 +1,49 @@
...
-      <TouchableOpacity style={styles.loginBtn} onPress={() => console.log('submit')}>
+      <Controller
+        name="email"
+        control={control}
+        render={({ onChange, value }) => <Form title="Email" value={value} onChange={onChange} />}
+        rules={{
+          required: { value: true, message: 'Campo requerido' },
+          validate: value =>
+            value.match(/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/) ||
+            'El email no es valido'
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Login/index.tsx]: válido

Comment (guidoprinc) [reply wbooks/src/app/screens/Login/index.tsx]: also always extract all validations inside a file in `utils` folder where the file should be called `validations.ts`

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/Login/components/Form/index.tsx (hunk 1 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Login/components/Form/index.tsx]: Done!

File: wbooks/src/app/screens/Login/index.tsx (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Login/index.tsx]: Done!

**Review by guidoprinc — APPROVED** [review]

**Pull request merged**